.github/workflows/test_pull_requests.yml

```diff
@@ -107,46 +107,51 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ ubuntu-latest ]
-        python: [ "3.9", "3.10" ]
```

Comment:
Is this `3.9` unchanged on purpose? I'm just scrolling through and notice this is the only one no bumped ^^

Reply:
I try to have working CI. And there is a PySide6/PySide2 problem